src/main/java/com/synopsys/integration/detect/lifecycle/run/data/DockerTargetData.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * synopsys-detect
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.detect.lifecycle.run.data;
+
+import java.io.File;
+import java.util.Optional;
+
+import org.jetbrains.annotations.Nullable;
+
+import com.synopsys.integration.detectable.Extraction;
+import com.synopsys.integration.detectable.detectables.docker.DockerExtractor;
+import com.synopsys.integration.util.NameVersion;
+
+public class DockerTargetData {
+
+    private NameVersion nameVersion;
+    @Nullable
+    File squashedImage;
+    @Nullable
+    File containerFilesystem;
+    @Nullable
+    File providedTar;
+
```

Comment:
ah, "provided" (instead of "given") also seems good. Whatever you pick, I'd suggest being consistent with naming of this thing between this class and RunResult.